modyn/selector/internal/selector_strategies/abstract_selection_strategy.py

```diff
@@ -358,9 +343,7 @@ def get_trigger_partition_keys(
             assert num_samples_trigger_partition is not None, f"Could not find TriggerPartition {partition_id} in DB"
             num_samples_trigger_partition = num_samples_trigger_partition[0]
 
-        data = TriggerSampleStorage(
-            self._trigger_sample_directory,
-        ).get_trigger_samples(
+        data = TriggerSampleStorage(self._trigger_sample_directory).get_trigger_samples(
             pipeline_id=self._pipeline_id,
             trigger_id=trigger_id,
             partition_id=partition_id,
```

Comment:
I think we might need to update the typing of `get_trigger_partition_keys`

Reply:
Done